docs/platforms/darwin/index.md

```diff
@@ -0,0 +1,11 @@
+# Darwin
```

Comment:
Thanks again for doing this, let me add a symlink in here, so the existing paths that public websites are using, can redirect to the new location

Reply:
Moving darwin back directly into the guides directory to avoid blocking this PR. Note that the handbook won't be linking the guides directory since it's a bit of a dumping ground, so if you want the darwin info to appear in the handbook you'll need to move this into a more sensible location with whatever simlinks you need.